README.md

```diff
@@ -0,0 +1,33 @@
+This is a fork of the https://github.com/apache/xalan-j project, intended for use in producing bug fix releases from upstream Xalan 2.7.1.
+
+How to build
+------------
+Switch to jdk 1.8
+
+Add EE 8 or earlier servlet and ejb API jars to the classpath.
+
+For example:
+
+    export CLASSPATH=~/.m2/repository/org/jboss/spec/javax/ejb/jboss-ejb-api_3.2_spec/2.0.0.Final/jboss-ejb-api_3.2_spec-2.0.0.Final.jar: \
+                     ~/.m2/repository/org/jboss/spec/javax/servlet/jboss-servlet-api_4.0_spec/2.0.0.Final/jboss-servlet-api_4.0_spec-2.0.0.Final.jar
```

Comment:
I happen to have javax.servlet api in:
~/.m2/repository/javax/servlet/javax.servlet-api/3.1.0/javax.servlet-api-3.1.0.jar
~/.m2/repository/javax/servlet/javax.servlet-api/4.0.1/javax.servlet-api-4.0.1.jar

Reply:
Everyone will have different jars I'm sure, especially if they occasionally delete their maven repo.  No change needed really.

Reply:
+1. 

I went with the spec fork GAVs as the example just because it's more likely a typical WildFly dev would have those vs having the javax.* Eclipse artifacts.